cli/cmd/cluster_test.go

```diff
@@ -599,15 +601,48 @@ var _ = Describe("GetClusterData()", func() {
 
 	Context("When command is running on the coordinator", func() {
 		BeforeEach(func() {
+			_ = os.Setenv("PXF_BASE", "../testdata")
 			_ = os.MkdirAll(coordinatorDataDir, 0777)
 		})
 
 		AfterEach(func() {
 			_ = os.RemoveAll(coordinatorDataDir)
 		})
 
+		It("returns an error when failing to get PXF_BASE or PXF_HOME", func() {
+			fakeSegConfigs = sqlmock.NewRows(header).AddRow(coordinator...).AddRow(localSegOne...).AddRow(localSegTwo...)
+			mock.ExpectQuery("SELECT (.*)").WillReturnRows(fakeSegConfigs)
```

Comment:
lets rename this mock since we have other mocks as well
```suggestion
			mockConnection.ExpectQuery("SELECT (.*)").WillReturnRows(fakeSegConfigs)
```